sign_oca/migrations/16.0.1.1.1/post-migration.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2024 ForgeFlow <http://www.forgeflow.com>
```

Comment:
This should be in pre, not post, or the column will already be created by the ORM and this will give error.